docs/backend/quantization.md

```diff
@@ -0,0 +1,102 @@
+# Quantization
+
+`SGLang` support various quantization methods, inclding online dynamic quantization and offline quantization.
+
+## Online Dynamic Quantization
+
+To enable oneline dynamic quantization, you can simply specify `--quantization` in command line. For example, if you want to enable `FP8` quantization for model `meta-llama/Meta-Llama-3.1-8B-Instruct`, you can lauch the server with following command:
+```bash
+python3 -m sglang.launch_server \
+    --model-path meta-llama/Meta-Llama-3.1-8B-Instruct \
+    --quantization fp8 \
+    --port 30000 --host 0.0.0.0
+```
+
+Then, you can test your FP8 quantized model with `curl`:
+```bash
+curl -s http://localhost:30000/v1/chat/completions \
+  -d '{"model": "meta-llama/Meta-Llama-3.1-8B-Instruct", "messages": [{"role": "user", "content": "What is the capital of France?"}]}'
+```
+
+<details>
+<summary>More quantization methods</summary>
+
+Our team is wokring on supporting more quantization methods with high priority. We will soon support other quantization methods including but not limitied to `["awq", "gptq", "marlin", "gptq_marlin", "awq_marlin", "bitsandbytes", "gguf"]`
+
+</details>
```

Comment:
This should not be folded by `<detail>`.

Reply:
Also, working but not wokring. Check the grammar. At least use GPT 😂

Reply:
Thanks. I will check all grammars with GPT agian. Sorry for these typos.